templates/CRM/common/customDataBlock.tpl

```diff
@@ -1,6 +1,6 @@
 
 {if !empty($customDataType)}
-  <div id="customData"></div>
+  <div id="customData_{$customDataType}"></div>
```

Comment:
In previous lives I think I've added js or css customizations that target this div. I'm probably not the only one who's done that. If this is now variable, it might be harder to target. Not sure if that's a blocker since I don't have a good sense of what's out there, but could we add a unique class like crm-customdata-container so at least it's an easy fix?

Reply:
that's a good idea - the participant class already had crm-customData-block so I made it that - we can also flag for release notes & maybe change log